docs/content/installation/uninstall_driver_openshift_web.md

```diff
@@ -20,4 +20,3 @@ Perform the following steps in order to uninstall the CSI driver and operator th
 <a name="operator"></a>5.  From the **Installed Operators** page, click on the **more** menu for the **Operator for IBM block storage CSI driver** and select **Uninstall Operator**.
```

Comment:
is 5. supposed to appear less indented than the rest of the steps?

Reply:
it's not. it has tagging before it. the file html and not-raw version looks fine

Reply:
it's a github processing error. i changed around my tagging so that it would look ok in github as well
done